actix-prost-convert-trait/src/lib.rs

```diff
@@ -9,6 +9,48 @@ mod bytes;
 #[cfg(feature = "conv-url")]
 mod url;
 
+#[cfg(feature = "conv-datetime")]
+mod datetime;
+
+#[cfg(feature = "conv-uuid")]
+mod uuid;
+
+#[cfg(feature = "conv-ip")]
+mod ip;
+
+#[cfg(feature = "conv-path")]
+mod path;
+
+#[cfg(feature = "conv-duration")]
+mod duration;
+
+#[cfg(feature = "conv-decimal")]
+mod decimal;
+
 pub trait TryConvert<T>: Sized {
     fn try_convert(value: T) -> Result<Self, String>;
 }
+
+#[macro_export]
+macro_rules! impl_try_convert_from_string {
+    ($type:ty, $type_name:expr) => {
+        impl TryConvert<String> for $type {
+            fn try_convert(value: String) -> Result<Self, String> {
+                value
+                    .parse()
+                    .map_err(|e| format!("failed to parse '{}' as {}: {}", value, $type_name, e))
```

Comment:
it might be easier and more precise to use `stringify!`, what do you think?

Reply:
oh, i remember. I dont like `PathBuf`, I wont to see `path` in error message

Reply:
will add second case like
```
($type:ty) => {
        impl_try_convert_from_string!($type, stringify!($type));
    };
```